vignettes/mouse.Rmd

```diff
@@ -104,3 +105,63 @@ betas = sesameDataGet("MM285.10.tissue")$betas
 visualizeGene("Igf2", betas = betas, platform="MM285", refversion = "mm10")
 ```
 
+## Infer Strain Information
+
+Let's load a pre-built `SigSet` object from SeSAMeData
+```{r}
+sset <- sesameDataGet('MM285.1.NOD.FrontalLobe')
+```
+
+Calculate beta values using the following commands.
+```{r}
+betas <- sset %>%
+         noob %>%
+         dyeBiasCorrTypeINorm %>%
+         getBetas
+```
+
+Convert the beta values to Variant Allele Frequencies.  
+It should be noted that since variant allele frequency is not always measured in green for Infinium-II and M-allele for Infinium-I, one needs to flip the beta values for some probes to calculate variant allele frequency.
+```{r}
+vafs <- betaToAF(betas)
+```
+
+Infer strain information for mouse array. This will return a list containing the best guess, p-value of the best guess, and probabilities of all strains.
+```{r}
+strain <- inferStrain(vafs)
+strain$pval
+```
+```{r echo=FALSE}
+library(ggplot2)
+df <- data.frame(strain=names(strain$probs), probs=strain$probs)
+ggplot(data = df,  aes(x = strain, y = log(probs))) +
+  geom_bar(stat = "identity", color="gray") +
+  ggtitle("strain probabilities") +
+  theme(axis.text.x = element_text(angle = 90), legend.position = "none")
+```
+
+## Contrast Data with Tissue References
+
+Let's load beta values from SeSAMeData
+```{r}
+betas <- sesameDataGet("MM285.10.tissue")$betas[,1:2]
+```
+
+Compare mouse array data with mouse tissue references. This will return a grid object that contrasts the traget sample with pre-build mouse tissue reference.
+```{r}
+compareMouseTissueReference(betas)
+```
+
+## Infer Mouse Age
+
+Let's load beta values from SeSAMeData
+```{r}
+betas <- sesameDataGet('MM285.10.tissue')$betas
+```
+
+The age of the mouse can be predicted using the `predicMouseAgeInMonth` function. This looks for overlapping probes and estimates age using an aging model built from 321 MM285 probes. The function returns a numeric output of age in months.  
```

Comment:
predicMouse is a typo.

Reply:
actually the model is using 347 probes. I should've updated the function description.